dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/data-storage.test.js

```diff
@@ -22,7 +25,253 @@ define([
+                    namespace: 'magento',
+                    search: '',
+                    filters: {
+                        store_id: 0
+                    },
+                    sorting: {},
+                    paging: {}
+                };
+
+                expect(model.getData(params)).toBeDefined();
+            });
+
+            it('check returned type if method called with argument', function () {
+                var params = {
+                    namespace: 'magento',
+                    search: '',
+                    filters: {
+                        store_id: 0
+                    },
+                    sorting: {},
+                    paging: {}
+                },
+                    type = typeof model.getData(params);
+
+                expect(type).toEqual('object');
+            });
+
+            it('check "clearRequests" has been called', function () {
+                var params = {
+                    namespace: 'magento',
+                    search: '',
+                    filters: {
+                        store_id: 0
+                    },
+                    sorting: {},
+                    paging: {}
+                };
+
+                spyOn(model, 'clearRequests');
+                spyOn(model, 'hasScopeChanged').and.callFake(function () {
+                    return true;
+                });
+                model.getData(params);
+                expect(model.clearRequests).toHaveBeenCalled();
+            });
+
+            it('check "getRequest" has been called', function () {
+                var params = {
+                    namespace: 'magento',
+                    search: '',
+                    filters: {
+                        store_id: 0
+                    },
+                    sorting: {},
+                    paging: {}
+                };
+
+                spyOn(model, 'getRequest');
+                spyOn(model, 'hasScopeChanged').and.callFake(function () {
+                    return false;
+                });
+                model.getData(params);
+                expect(model.getRequest).toHaveBeenCalled();
+            });
+
+            it('Return "getRequestData" method', function () {
+                var params = {
+                    namespace: 'magento',
+                    search: '',
+                    filters: {
+                        store_id: 0
+                    },
+                    sorting: {},
+                    paging: {}
+                },
+                    options = {
+                        refresh: false
+                    };
+
+                spyOn(model, 'getRequestData');
+                spyOn(model, 'getRequest').and.callFake(function () {
+                    return true;
+                });
+                model.getData(params, options);
+                expect(model.getRequestData).toHaveBeenCalled();
+            });
+
+            it('Return "requestData" method', function () {
```

Comment:
Along the same lines as my last comment, let's qualify the test name further. How does it differentiate from the previous test? having multiple tests with the same name is not recommended - that can be quite confusing. Wording matters! Remember that tests are a tool to help us, the developers, be productive and guide us. Spending  the extra few minutes to properly comment, document and name things goes a long way to making the code maintainable long term.